src/test/skript/tests/syntaxes/expressions/ExprItemWithEnchantmentGlint.sk

```diff
@@ -0,0 +1,43 @@
+test "item with enchantment glint" when running minecraft "1.20.5":
+
```

Comment:
Thoughts about adding some test using the parse section testing most if not all patterns?

Reply:
Resolved - dealt with suggestion author this is not necessary